### DISCUSSION TITLE
Suggestion - Per-ruleset custom descriptions

### DISCUSSION THREAD

**Discussion by dunsparce9** [ppy/osu#26745] [Ideas]
![image](https://github.com/ppy/osu/assets/66077935/1544ade6-91db-4ae9-8659-2ced39f17066)

The generic "play some X" description can get tiring if you have a lot of custom rulesets loaded.
This was probably suggested before but it would be nice if the `RulesetInfo` got a `Description` field which rulesets could then use to add in their little quips when you hover their icon in the top bar.

Some i came up with:
osu! - click some circles
osu!taiko - play some drums
osu!catch - collect some fruit
osu!mania - smash some keys
etc.

**Comment by marioman63** [comment]
agreed. as it stands, it feels a bit redundant to have that flavour text if that's all it displays.
